packages/typespec-ts/package.json

```diff
@@ -62,7 +62,7 @@
     "@azure-tools/cadl-ranch-expect": "^0.9.0",
     "@azure-tools/cadl-ranch": "^0.10.0",
     "chalk": "^4.0.0",
-    "@azure-rest/core-client": "^1.1.6",
+    "@azure-rest/core-client": "/Users/zhangqiaoqiao/work/code/azure-sdk-for-js/sdk/core/core-client-rest/azure-rest-core-client-1.2.0.tgz",
```

Comment:
we will need a new release of `@azure-rest/core-client` once https://github.com/Azure/azure-sdk-for-js/pull/27343 is merged.